artifacts/src/main/resources/common/protocol-version-schema.json

```diff
@@ -22,6 +22,12 @@
                             },
                             "path": {
                                 "type": "string"
+                            },
+                            "binding": {
+                                "type": "string"
+                            },
+                            "auth": {
+                                "type": "string"
                             }
                         },
                         "required": ["version", "path"]
```

Comment:
Just asking: I can follow that `auth` is optional as we can have unprotected endpoints... why do we allow versions without bindings; are there any cases? If not, I would make it a mandatory property.